### PR TITLE
bacpop-143 update for GPS_v6

### PR DESCRIPTION
**Pull request by EmmaLRussell**
Updates config for GPS_v6 database

Also fixes dev configuration by getting correct OAuth app details from vault, and setting redis url. 
You should be able to run this configuration locally with: `./beebop start --pull dev` 
The app should be available at https://localhost

There's a separate ticket to add a working staging configuration: https://mrc-ide.myjetbrains.com/youtrack/issue/bacpop-150

### CODE REVIEW

File: config/beebop.yml [hunk 1]
```diff
@@ -13,14 +13,15 @@ server:
   image:
     repo: mrcide
     name: beebop-server
-    tag: main
+    tag: bacpop-145
```
Comment (EmmaLRussell) [inline config/beebop.yml]: To be updated when this branch merged

File: config/beebop.yml [hunk 2]
```diff
@@ -13,14 +13,15 @@ server:
...
   port: 4000
+  redis_url: "redis://beebop-redis:6379"
```
Comment (EmmaLRussell) [inline config/beebop.yml:18]: The default value in the server was failing as it was pointing to localhost

File: test/test_integration.py [hunk 3]
```diff
@@ -26,19 +26,19 @@ def test_start_beebop():
 
     assert docker_util.network_exists("beebop_nw")
     assert docker_util.volume_exists("beebop_storage")
-    assert docker_util.container_exists("beebop_api")
-    assert docker_util.container_exists("beebop_redis")
-    assert docker_util.container_exists("beebop_server")
-    assert docker_util.container_exists("beebop_proxy")
-    assert len(docker_util.containers_matching("beebop_worker_", False)) == 2
+    assert docker_util.container_exists("beebop-api")
```
Comment (EmmaLRussell) [inline test/test_integration.py:29]: I'm guessing these container names have changed in later versions of constellation..?

Comment (richfitz) [reply test/test_integration.py:29]: yeah, we moved to hyphens at some point - I forget why

**Review by richfitz — APPROVED** [review]

**Pull request merged**